packages/client-react-streaming/src/DataTransportAbstraction/WrappedApolloClient.test.tsx

```diff
@@ -55,12 +55,12 @@ describe(
     };
     const FIRST_RESULT = { me: "User" };
     const EVENT_DATA: QueryEvent = {
-      type: "data",
+      type: "next",
```

Comment:
changing from the old `QueryEvent` type to the new one that is an extension of `ReadableStreamLinkEvent`